app/packages/utilities/src/fetch.ts

```diff
@@ -89,11 +89,11 @@ export const setFetchFunction = (
     const fetchCall = retries
       ? fetchRetry(fetch, {
           retries,
-          retryDelay,
+          0,
           retryOn: (attempt, error, response) => {
             if (
               error !== null ||
-              (response.status >= 400 && attempt < retries)
```

Comment:
Wait I'm confused; isn't this PR reducing the number circumstances in which retries are performed?

Reply:
Yes. But why would we want to retry in all of those scenarios?

We retry on the event source indefinitely to check for liveliness, but we really shouldn't be retrying on other requests. An error means something isn't working...

Reply:
I was asking bc I didn't see how this PR was adding anything, but I missed that `retries` used to be 0